recipes/_r-mutex/LICENSE.txt

```diff
@@ -1,14 +0,0 @@
-BSD 3-clause license
```

Comment:
No reason to remove other recipes. Please restore

Reply:
@scopatz I am sorry about that, I thought I could only have one recipe when doing the pull request so I took r-mutex out. I am not sure how to restore it. I put r_mutex recipe back into my staged-recipe folder and pushed Polycircles again. I am hoping that does it. Any guidance is welcome! Best.

Reply:
I think if you merge from the upstream master, this should fix it.

Reply:
@scopatz thanks!